### PR TITLE
Connects frontend components to localStorage + jubSignal logic

### PR DESCRIPTION
**Pull request by AndrewCLu**
- [x] Populate activity feed and contacts list with localStorage data
- [x]  Connect users/[id] page with localStorage data
- [x] Update styling of users/[id]/share page
- [x] Make activity feed and contacts list click to user and location pages
- [x] Update timestamp to store as ISO strings

### CODE REVIEW

File: src/pages/users/[id]/share.tsx
Comment (RiverRuby) [inline src/pages/users/[id]/share.tsx:172]: We need frontends for this image.png

Comment (RiverRuby) [reply src/pages/users/[id]/share.tsx:172]: image

File: src/pages/users/[id]/share.tsx
Comment (RiverRuby) [inline src/pages/users/[id]/share.tsx:174]: Back button doesn't work here

File: src/pages/users/[id]/share.tsx
Comment (RiverRuby) [inline src/pages/users/[id]/share.tsx:204]: Not able to send data, not sure what my error is

File: src/pages/users/[id]/index.tsx
Comment (RiverRuby) [inline src/pages/users/[id]/index.tsx:42]: This should only appear if you haven't already connected right? Not by default?

File: src/pages/users/[id]/index.tsx
Comment (RiverRuby) [inline src/pages/users/[id]/index.tsx:66]: We should make the input field during registration/profile update for twitter not say "twitter.com/username" it should start with an "@" and then just have a prompt for user name so this actually works.

File: src/pages/users/[id]/share.tsx
Comment (RiverRuby) [inline src/pages/users/[id]/share.tsx:183]: Definitely confusing that this alternates betwen `twitterUsername` and `x` for different objects

File: src/pages/locations/[id].tsx
Comment (RiverRuby) [inline src/pages/locations/[id].tsx:58]: This leads to an infinite loop where it takes you back to the tap page and it says you've already visited. This should link directly to social tab

File: src/pages/index.tsx
Comment (RiverRuby) [inline src/pages/index.tsx:88]: Location tap isn't showing up in my feed

Comment (RiverRuby) [reply src/pages/index.tsx:88]: jk upon login it appears but it's duplicated 
image

**Review by RiverRuby — COMMENTED** [review]
I think there's a few bugs that need to be resolved before approval.

File: src/pages/users/[id]/share.tsx
Comment (AndrewCLu) [reply src/pages/users/[id]/share.tsx:183]: The rule is x is used whenever space matters and twitterUsername is used everywhere else, to be more descriptive.

File: src/pages/users/[id]/index.tsx
Comment (AndrewCLu) [reply src/pages/users/[id]/index.tsx:42]: I think this is a design discussion - i'm thinking its totally valid for people to reshare if they want to say update the information they send to someone, but we could also disable this

File: src/pages/users/[id]/share.tsx
Comment (AndrewCLu) [reply src/pages/users/[id]/share.tsx:209]: Error was due to not calling `event.preventDefault()` on the form, oops...

File: src/pages/locations/[id].tsx
Comment (AndrewCLu) [reply src/pages/locations/[id].tsx:58]: https://github.com/nfc-denver/nfc-denver/issues/68

File: src/pages/index.tsx
Comment (AndrewCLu) [reply src/pages/index.tsx:88]: This is due to React double-triggering effects in dev environment on strict mode. We can either leave this (it has no effect on app logic) or add a ref that addresses this

File: src/pages/users/[id]/index.tsx
Comment (AndrewCLu) [reply src/pages/users/[id]/index.tsx:66]: I think this is a good thing to bringup during the design review - we should definitely have a list of these small frontend fixes that kali can just knock out all at once

File: src/pages/users/[id]/share.tsx
Comment (AndrewCLu) [reply src/pages/users/[id]/share.tsx:177]: https://github.com/nfc-denver/nfc-denver/issues/29

**Review by RiverRuby — APPROVED** [review]
LGTM, login for server back up is borked but i made an issue https://github.com/nfc-denver/nfc-denver/issues/70

**Review by RiverRuby — APPROVED** [review]
LGTM, login for server back up is borked but i made an issue https://github.com/nfc-denver/nfc-denver/issues/70

**Pull request merged**
